src/asfvideo.cpp

```diff
@@ -21,149 +25,177 @@ namespace Exiv2::Internal {
   - https://fr.wikipedia.org/wiki/Advanced_Systems_Format
   - https://exse.eyewated.com/fls/54b3ed95bbfb1a92.pdf
  */
-const std::map<std::string, std::string> GUIDReferenceTags = {
+/*
+ * @class GUID_struct
+ *
+ * @brief A class to represent a globally unique identifier (GUID) structure
+ *
+ * This class represents a globally unique identifier (GUID) structure which is used to identify objects in a
+ * distributed environment. A GUID is a unique identifier that is generated on a computer and can be used to
+ * identify an object across different systems. The GUID structure is comprised of four 32-bit values and an
+ * array of 8 bytes.
+ *
+ * @note The byte order of the GUID structure is in little endian.
+ *
+ * @see https://en.wikipedia.org/wiki/Globally_unique_identifier
+ *
+ */
+
+bool AsfVideo::GUIDTag::operator==(const AsfVideo::GUIDTag& other) const {
```

Comment:
This is equivalent to = default. No need for this.

Reply:
I beleive you are talking about operator= but here I developped operator== to compare with other tags.
```
github/exiv2/src/asfvideo.cpp:203:17: error: invalid operands to binary expression ('const AsfVideo::GUIDTag' and 'AsfVideo::GUIDTag')
  return Header == AsfVideo::GUIDTag(buf);
```